src/frontend/src/components/mainWindow.ts

```diff
@@ -0,0 +1,53 @@
+import { html, TemplateResult } from "lit-html";
+import { ifDefined } from "lit/directives/if-defined.js";
+import { icLogo } from "./icons";
+import { footer } from "./footer";
+
+export type mainWindowProps = {
+  slot: TemplateResult;
+  uid?: string;
```

Comment:
```suggestion
  id?: string;
```
nit: `uid` is not used anywhere else in the codebase